src/coreclr/tools/Common/TypeSystem/Common/FieldDesc.FieldLayout.cs

```diff
@@ -11,7 +11,7 @@ public partial class FieldDesc
     {
         private LayoutInt _offset = FieldAndOffset.InvalidOffset;
 
-        public LayoutInt Offset
+        public virtual LayoutInt Offset
```

Comment:
I was trying to avoid having these custom `FieldDesc` types percolate through the whole type system. I'll see if I can come up with something that keeps the model tightly scoped to the cases where it matters.